app/admin/practices.rb

```diff
@@ -5,14 +5,110 @@
 
   scope :published
   scope :unpublished
+  scope :Get_Practice_Owner_Emails
```

Comment:
the ruby way: let's lower case this

Reply:
Got it... i've updated both of those instances to lower case and have pushed.